docs/crystal_for_rubyists/README.md

```diff
@@ -404,6 +404,46 @@ private def method
 end
 ```
 
+### Hash syntax from Ruby to Crystal
+
+Crystal introduces a data type that is not available in Ruby, the NamedTuple.
+
+Typically in Ruby you can define a hash with several syntaxes:
+
+```ruby
+# A valid ruby hash declaration
+{ 
+  key1: "some value",
+  some_key2: "second value"
+}
+
+# This syntax in ruby is shorthand for the hash rocket => syntax
+{
+  :key1 => "some value",
+  :some_key2 => "second value"
+}
+```
+
+In Crystal, this is not the case. The hash rocket `=>` syntax is required to declare a hash in Crystal.
+
+```crystal
+# Creates a valid `Hash(Symbol, String)` in Crystal
+{
+  :key1 => "some value",
+  :some_key2 => "second value"
+}
+
+# Creates a NamedTuple in Crystal
+{ 
+  key1: "some value",
+  some_key2: "second value"
+}
+```
+
+However, the hash shorthand syntax in Ruby creates a [NamedTuple](https://crystal-lang.org/api/latest/NamedTuple.html) in Crystal.
```

Comment:
This introductory paragraph should better be placed *above* the example that demonstrates it.

Reply:
I moved this up above the example as suggested